common/test/services/prismic/parsers.test.js

```diff
@@ -9,16 +12,45 @@ export const sameAs = [
     title: [{ type: 'heading1', text: 'This is it!', spans: [] }],
   },
 ];
-it('should parse same as correctly', () => {
-  const [twitter, url, urlWithTitleOverride] = parseSameAs(sameAs);
 
-  expect(twitter).toEqual({
-    link: 'https://twitter.com/mbannisy',
-    title: '@mbannisy',
+describe('parsers', () => {
+  describe('parseSameAs', () => {
+    it('should parse same as correctly', () => {
+      const [twitter, url, urlWithTitleOverride] = parseSameAs(sameAs);
+      expect(twitter).toEqual({
+        link: 'https://twitter.com/mbannisy',
+        title: '@mbannisy',
+      });
+      expect(url).toEqual({ link: 'http://things.com', title: 'things.com' });
+      expect(urlWithTitleOverride).toEqual({
+        link: 'https://google.com',
+        title: 'This is it!',
+      });
+    });
   });
-  expect(url).toEqual({ link: 'http://things.com', title: 'things.com' });
-  expect(urlWithTitleOverride).toEqual({
-    link: 'https://google.com',
-    title: 'This is it!',
+
+  describe('parseMediaObjectList', () => {
+    const mockData = [
+      {
+        content: {
+          id: 'X5BeNxIAACEAj7Ze',
+          type: 'media_object',
+          tags: [],
+          slug: 'wear-a-face-covering-hi',
+          lang: 'en-gb',
+          link_type: 'Document',
+          isBroken: false,
+          // data: [] -> this is missing from data structure
+          data: {
+            broken: 'data',
```

Comment:
My concern here is that this is an unreachable state given it comes from Prismic. It can definitely be missing, which we should account for, but it cannot be broken.

Reply:
I will refactor the parser as you mentioned previously.